advocacy_docs/supported-open-source/postgresql/installing/windows.mdx

```diff
@@ -27,10 +27,12 @@ Rather than use the EDB installer, you can also obtain a prebuilt installation p
 
 ## Installing PostgreSQL
 
-To perform an installation using the graphical installation wizard, you need superuser or administrator privileges. 
+To perform an installation using the graphical installation wizard, you need superuser or administrator privileges.
 
-!!! Note
-    If you're using the graphical installation wizard to perform a system upgrade, the installer preserves the configuration options specified during the previous installation.
+!!!note Notes
+   - Multi-byte, non-ascii characters are not supported in user or machine names.
```

Comment:
```suggestion
   - EDB doesn't support all non-ASCII, multi-byte characters in user or machine names. Use ASCII characters only to avoid installation errors related to unsupported path names.
```